design-tokens/tokens/layer1/base.json

```diff
@@ -116,6 +116,48 @@
           "value": "25rem",
           "type": "dimension"
         }
+      },
+      "height": {
+        "100": {
+          "value": "0.25rem",
+          "type": "dimension"
+        },
+        "200": {
+          "value": "0.5rem",
+          "type": "dimension"
+        },
+        "400": {
+          "value": "1rem",
+          "type": "dimension"
+        },
+        "500": {
+          "value": "1.25rem",
+          "type": "dimension"
+        },
+        "600": {
+          "value": "1.5rem",
+          "type": "dimension"
+        },
+        "800": {
+          "value": "2rem",
+          "type": "dimension"
+        },
+        "900": {
+          "value": "2.25rem",
+          "type": "dimension"
+        },
+        "1000": {
+          "value": "2.5rem",
+          "type": "dimension"
+        }
+      },
+      "negative": {
+        "spacing": {
+          "100": {
+            "value": "-0.063rem",
```

Comment:
you want 1 px ?

Reply:
we can change to -1px instead of the -0.063rem

Reply:
Yeah, I don't think negative spacing like this is related to font size